File: 3.Operaciones_Logicas/java/main.java
import java.lang.Math;

public class Main{
    public static void main(String[] args) { 
        int numberOne=4, numberTwo=5;
        int resultSum= numberOne+numberTwo;
        int resultRest= numberOne-numberTwo;
        int resultMult= numberOne*numberTwo;
        float resultDiv= (float)(numberOne/numberTwo); // Casteo
        int resultMod= numberOne%numberTwo;
        int resultExp= (int)(Math.pow(numberOne, numberTwo)); //Casteo
        int resultSqrt= (int)(Math.sqrt(numberOne)); //Casteo 

        System.out.println("Suma "+ resultSum);
        System.out.println("Resta "+ resultRest);
        System.out.println("Multiplicacion "+ resultMult);
        System.out.println("Division "+ resultDiv);
        System.out.println("Modulo "+ resultMod);
        System.out.println("Exponencia "+ resultExp);
        System.out.println("Raiz "+ resultSqrt);
    }
}

File: 3.Operaciones_Logicas/c/main.c
#include <stdlib.h>
#include <stdio.h>
#include <math.h>

int main()
{
    int numberOne=4, numberTwo=5;
    int resultSum= numberOne+numberTwo;
    int resultRest= numberOne-numberTwo;
    int resultMult= numberOne*numberTwo;
    float resultDiv= (float)(numberOne/numberTwo); // Casteo
    int resultMod= numberOne%numberTwo;
    int resultExp= pow(numberOne, numberTwo);
    int resultSqrt= sqrt(numberOne);

    printf("Suma %d \n", resultSum);
    printf("Resta %d \n", resultRest);
    printf("Multiplicacion %d \n", resultMult);
    printf("Division %d \n", resultDiv);
    printf("Modulo %d \n", resultMod);
    printf("Exponencia %d \n", resultExp);
    printf("Raiz %d \n", resultSqrt);
}